app/import.go

```diff
@@ -38,7 +38,7 @@ func stopOnError(err LineImportWorkerError) bool {
 	}
 }
 
-func processAttachmentPaths(files *[]AttachmentImportData, basePath string, filesMap map[string]*zip.File) error {
+func processAttachmentPaths(files *[]*AttachmentImportData, basePath string, filesMap map[string]*zip.File) error {
```

Comment:
```suggestion
func processAttachmentPaths(files []*AttachmentImportData, basePath string, filesMap map[string]*zip.File) error {
```
I don't think there's a need to pass pointer to array here.